AutoDraft/data_exploration.py

```diff
@@ -0,0 +1,257 @@
+import json, copy 
+import streamlit as st
+import numpy as np 
+import pandas as pd
+from pandas.io.json import json_normalize
+import requests as rqsts
+# import plotly.express as px
+
+st.title('NHL API Dataset Assembly')
```

Comment:
I don't know how much you'll reuse this exploration script, but I'd recommend not interlacing functions and function calls, and instead move the function calls inside an `if __name__ == '__main__':` statement. Maybe this will conflict with streamlit or something though...